js/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const pug = require('pug');
+const log = require('./log');
+const router = require('./route');
+
+const app = express();
+app.set('view engine', 'pug');
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(router);
+
+app.listen(8008, () => {
```

Comment:
export port number into const, please
provide user to set port number by npm script args, please